integration-tests/integration_tests/test_scenarios/int_smoke_tests.py

```diff
@@ -1,20 +1,35 @@
 from unittest import skip, TestCase
-
-from integration_tests.helpers import methods, interactions, message_retriever
+from integration_tests.helpers import methods, interactions, message_retriever, xml_parser
 
 
 class FunctionalTest(TestCase):
+    # Message Pattern Type: Asynchronous Express
+    # Interaction: PSIS Document List Data Request (QUPC_IN160101UK05)
+    def test_mhs_async_express(self):
```

Comment:
this test is not needed, as we have a seperate file for these now in integration tests, using the new pattern for testing.

Reply:
Done